### ISSUE TITLE
Can not let a tamed cat stand without a fish

### ISSUE THREAD

**Issue by Gardelll** [ProtocolSupport/ProtocolSupport#1256]
> * Ensure you are using the latest version of ProtocolSupport from Jenkins: https://build.true-games.org/job/ProtocolSupport/ (`/ps buildinfo`) 

YES

> * Ensure you are using the latest version of Spigot, PaperSpigot or Glowstone as appropriate

YES

> * Make sure the problem is actually related to ProtocolSupport - try to reproduce it using only ProtocolSupport

YES

> When posting the issue you also need to provide the following information:

```
* ProtocalSupport: buildtime: 2020.07.01 at 03:45:14 MSK, buildhost: build.true-games.org, buildnumber: 504, buildgit: https://github.com/ProtocolSupport/ProtocolSupport:origin/master:af30c85d3334be6f0001bcbd51fff75126dc22f8
* Server version: git-Paper-38 (MC: 1.16.1)
```

------------ 

Try to right click to stand up a tamed cat without a fish, then "PlayerInteractEntityEvent" will fire but the cat will not stand.
I can see the PlayerInteractEntityEvent fired twice, is that the reason?

**Comment by Shevchik** [comment]
Affect client versions?

**Comment by Gardelll** [comment]
> 
> 
> Affect client versions?

I am using Vanilla Minecraft 1.16.1

**Comment by Shevchik** [comment]
Fixed for 1.15+ and 1.8- in https://github.com/ProtocolSupport/ProtocolSupport/commit/e2836d4d5737e70489c9cafc32b7e44ce4811d05

Tameable entity interaction issue for other versions is now tracked here: https://github.com/ProtocolSupport/ProtocolSupport/issues/1258
